package.json

```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "react-test-renderer": "^17.0.2",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
```

Comment:
- [OPTIONAL] Using this option `"test": "react-scripts test --verbose",` helps seeing your tests description.

Reply:
Thank you for this suggestion. I will do it.